frontend/src/components/SelectColumn/SelectColumn.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+interface SelectColumnProps {
+  id: string;
+}
+
+const SelectColumn: FunctionComponent<SelectColumnProps> = (props) => {
+  const { attributes, listeners, setNodeRef, transform, transition } = useSortable({
+    id: props.id,
+  });
```

Comment:
I think you can have this on a single line.

Reply:
Prettier complains when I make it a one liner